nss-c31-mixed-havoc/nutshell
# creating htmlFactory object with 2 functions to createElement and cle…

…arContainer

# Description

Created new js file with one object HTMLFactory. The object has two functions. One is createElementWithText which is a function to build an element taking three parameters. Second function is clearContainer which clears the parent container of the element which is the argument. Created loginScripts subfolder in scripts and 3 empty js files which to later add to.

Applies to issue #6

## Type of change

# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-html-factory`
3. cd into src/lib and rung `grunt`
4. verify that createElementWithText is console logging and h1 element


# Checklist:

- [x ] My code follows the style guidelines of this project
